Gemfile

```diff
@@ -100,10 +100,13 @@ end
 
 group :test do
   gem "capybara", "~> 2.14.3"
-  gem "codeclimate-test-reporter", group: :test, require: nil
+  gem "capybara-screenshot", "~> 1.0.0"
+  gem "chromedriver-helper"
+  gem "codeclimate-test-reporter", require: false
```

Comment:
Why did you remove the `test` group here ?

Reply:
Because it's already inside of the test group.